america/mexico/fiscal.source-spec.yaml

```diff
@@ -19,159 +21,197 @@ sources:
   url: http://www.transparenciapresupuestaria.gob.mx/work/models/PTP/DatosAbiertos/BD_Cuenta_Publica/CSV/cuenta_publica_2015_ra_ecd_epe.csv
 - encoding: windows-1252
   url: http://www.transparenciapresupuestaria.gob.mx/work/models/PTP/DatosAbiertos/Bases_de_datos_presupuesto/CSV/pef_ac01_avance_2016.csv
-
+- encoding: windows-1252
+  url: http://www.transparenciapresupuestaria.gob.mx/work/models/PTP/DatosAbiertos/Bases_de_datos_presupuesto/CSV/pef_ac01_2017.csv
 
 fields:
 # Fiscal year
 - header: CICLO
+  title: Ciclo
   aliases:
   - Ciclo
   options: {}
   osType: date:fiscal-year
 
 # Functional Classification
 - header: GPO_FUNCIONAL
+  title: Grupo Funcional
   aliases:
   - Grupo Funcional
   - Finalidad
   - GF
+  - GPO_FUNCIONAL
```

Comment:
You only need to add aliases if you change the header name to something different - you don't need to duplicate: (i.e. writing GPO_FUNCIONAL in the header is enough, no need to write it in an alias as well)